backend/internal/storage/storage_test.go

```diff
@@ -0,0 +1,23 @@
+package storage
+
+import (
+	"testing"
+
+	"github.com/ducanhpham0312/zeevision/backend/internal/environment"
+	"github.com/stretchr/testify/assert"
+)
+
+const DSN = "user=user password=password dbname=zeevision_db host=postgres port=5432"
+
+func TestNewDsn(t *testing.T) {
+	dsnConfig := DsnConfig{
+		User:         "user",
+		Password:     "password",
+		DatabaseName: environment.DatabaseName(),
+		Host:         environment.HostDatabase(),
+		Port:         environment.DatabasePort(),
```

Comment:
You can use hard-coded strings here since this is a test and shouldn't depend on the context.